M2/Macaulay2/packages/SimpleDoc/doc.txt

```diff
@@ -16,20 +16,20 @@ Node
 
     Tree
       :Getting started with SimpleDoc
-        @TO2 {"Macaulay2Doc :: writing documentation", "Writing documentation in Macaulay2"}@
-        @TO2 {doc, "Using SimpleDoc for documentation"}@
-        @TO2 {"Text", "Using markup in documentation"}@
-        @TO2 {"Text::html(TEX)", "Using TeX in documentation"}@
+       - @TO2 {"Macaulay2Doc :: writing documentation", "Writing documentation in Macaulay2"}@
+       > @TO2 {doc, "Using SimpleDoc for documentation"}@
+       - @TO2 {"Text", "Using markup in documentation"}@
```

Comment:
What is the difference between - and > ?  Can I use : in general, when I'm not creating a Tree?  Does Tree automatically create Node's now?